src/cpp/number-9.cpp

```diff
@@ -0,0 +1,14 @@
+#include <iostream>
+#include <random>
+
+int main() {
+    srand(time(NULL));
+
+    if (rand() % 9 + 1 == 9) {
+        std::cout << "Number 9...";
```

Comment:
Lines should end with a newline 

```suggestion
        std::cout << "Number 9...\n";
```

Can you add them for both outputs?